packages/common/src/discussions/api/types.ts

```diff
@@ -181,6 +181,15 @@ export enum CommonSort {
   UPDATED_AT = "updatedAt",
 }
 
+/**
+ * @export
+ * @enum {string}
+ */
+export enum SearchPostsFormat {
+  CSV = "CSV",
```

Comment:
For consistency with AGO, let's change the value of these to lowercase https://developers.arcgis.com/rest/users-groups-and-items/common-parameters/